system_tests/bigquery.py

```diff
@@ -90,7 +92,12 @@ def test_update_dataset(self):
         after = [grant for grant in dataset.access_grants
                  if grant.entity_id != 'projectWriters']
         dataset.access_grants = after
-        dataset.update()
+
+        @retry(Forbidden, tries=3, delay=30)
```

Comment:
Just manual trial and error and trying to keep them as low as I could. I'll add the comment!